WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift

```diff
@@ -48,8 +48,8 @@ extension LoginEpilogueUserInfo {
 
     /// Updates the Epilogue properties, given a SocialService instance.
     ///
```

Comment:
I noticed this comment being outdated as it mentions the `SocialService` which is now `SocialUser`.

Thinking about it, I don't think it adds any information that a reader wouldn't already gather from the method signature.

```suggestion
```